Medborg Crew Monitor Upgrade

Surrealaser opened the pull request:
### About The Pull Request
Integrates an item action for remote crew monitor access with the crew locator upgrade for the medborg.

### Why It's Good For The Game
The important second part of the crew pinpointer; as a result, medborgs no longer have to glue themselves to fixed crew monitor consoles in order to do the paramedicine job they excel at, which is boring and lame.

### Changelog
🆑
add: Remote access to crew monitoring is now integrated with the crew locator upgrade for Mediborgs.
/🆑

Surrealaser committed:
Medborg Crew Monitor Upgrade

Adds the crew monitor to medborg upgrades.

Why This is Good for the Game: The important second part of the crew pinpointer; as a result, medborgs no longer have to glue themselves to fixed crew monitor consoles in order to do the paramedicine job they excel at, which is boring and lame.

carlarctg commented:
can't you just merge this with the crew pinpointer upgrade? there's already like four other upgrade modules solely for the medborg, and it's getting kind of dumb and is a hassle to put into the borg, too

Surrealaser commented:
@carlarctg I actually might do just that. I agree, the upgrade spam is kind of obnoxious. I actually wanted to include this as a default property of the medborg but figured that might be too controversial.

Surrealaser committed:
Update

Revised per request so that the crew monitor upgrade is integrated with the pin pointer.

Surrealaser commented:
@carlarctg Changed it per your request.

nemvar commented on code/game/objects/items/robot/robot_upgrades.dm:
```diff
@@ -561,31 +561,49 @@
-		if (PP)
-			R.module.remove_module(PP, TRUE)
+		R.module.remove_module(PP, TRUE)
+		to_chat(world, "<span class='warning'>crew_monitor: [crew_monitor].</span>")
```
Delet dis

Surrealaser replied:
Ah fuck. Forgot about that debug. Thanks.

Surrealaser replied:
Fixed

Surrealaser committed:
Debug removal

ExcessiveUseOfCobblestone commented:
button only no verb

ExcessiveUseOfCobblestone commented:
really could just be a portable crew monitor module tho

Surrealaser commented:
@ExcessiveUseOfCobblestone Too clumsy to merely add it as a PCM IMO. But sure, I can remove the verb.

ExcessiveUseOfCobblestone commented:
what do you mean too clumsy...?

Surrealaser commented:
@ExcessiveUseOfCobblestone Yet another object among many to juggle. Personally I would prefer it to be hands free for QoL purposes, and it makes sense that you have a direct remote connection to the crew monitor information as a borg. Further, it makes sense to integrate it with the pinpointer as it an integral pair that need each other. Having it available as a separate upgrade adds to the obnoxious upgrade spam mentioned by @carlarctg . 

The only way I wouldn't mind this is if Borgs started off with a hand held monitor by default.

ExcessiveUseOfCobblestone commented:
I can't remember off the top of my head but if PCM lose the UI if you put it in say your pocket then medborgs should have this restriction as well imo since the only thing that balances this feature is the fact that you have to constantly stare at the thing.

If not then sure that's fine

Surrealaser commented:
@ExcessiveUseOfCobblestone Not sure, but even if the UI went away when stashed in the pocket, having it as a non-default researchable upgrade is IMO adequate downside for not needing it in the hand. I mean it's not like there is a substantial benefit from being able to see it in real time, versus taking it out as needed; the difference is mainly QoL.

Surrealaser committed:
Verb removal

Names-Are-Hard commented:
Portable crew monitors do work when pocketed. @ExcessiveUseOfCobblestone

Surrealaser commented:
@nemvar Is this ready to go?

ExcessiveUseOfCobblestone commented:
> @ExcessiveUseOfCobblestone Not sure, but even if the UI went away when stashed in the pocket, having it as a non-default researchable upgrade is IMO adequate downside to balance out not needing it in the hand. I mean it's not like there is a substantial benefit from being able to see it in real time, versus taking it out as needed; the difference is mainly QoL.

QoL is balance for this dumb feature (crew monitors in general, not what ur coding now), since it's literally only balanced by the ease of checking the thing and continually checking it, which something a borg player is nuts enough to do :omegalol:
